webhooks/models.py

```diff
@@ -0,0 +1,98 @@
+from typing_extensions import Literal
+from pydantic import ValidationError
+
+from shared_models import (
+    Connection,
+    PresentationExchange,
+    CredentialExchange,
+    HookBase,
+)
+
+
+class ConnectionsHook(HookBase, Connection):
+    pass
+
+
+def to_connections_model(item: dict) -> ConnectionsHook:
+    if item["connection_protocol"] == "didexchange/1.0":
+        item["connection_id"] = "0023-" + item["connection_id"]
+    elif item["connection_protocol"] == "connections/1.0":
+        item["connection_id"] = "0016-" + item["connection_id"]
```

Comment:
We don't prefix the connection id in the cloud api, so to stay consistent I think we also shouldn't do it for the webhooks (they should match 1:1)

The reason we did this for credentials / proofs is that they have separate records under the hood for v1 and v2. But with connections it is the same record for didexchange and connections